src/main/java/CellContent.java

```diff
@@ -0,0 +1,97 @@
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+
+public class CellContent {
+	public CellType type;
+	public boolean bool;
+	public double numeric;
+	public String string;
+	
+	public boolean wasFormula;
+	
+	public CellContent(Cell cell) {
+		if(cell == null) {
+			type = CellType.BLANK;
+			return;
+		}
+		
+		CellType temp_type = cell.getCellTypeEnum();
+		
+		wasFormula = false;
+		if(temp_type == CellType.FORMULA) {
+			temp_type = cell.getCachedFormulaResultTypeEnum();
+			wasFormula = true;
+		}
+		
+		type = temp_type;
+		switch(type) {
+			case BLANK:
+				break;
+				
+			case BOOLEAN:
+				bool = cell.getBooleanCellValue();
+				break;
+				
+			case NUMERIC:
+				numeric = cell.getNumericCellValue();
+				break;
+				
+			case STRING:
+				string = cell.getStringCellValue();
+				break;
+				
+			default:
+				break;
+		}
+	}
+	
+	public CellContent(String content) {
+		try { 
+			int numeric_value = Integer.parseInt(content);
+			type = CellType.NUMERIC;
+			numeric = numeric_value;
+		} catch(NumberFormatException e) {
+			type = CellType.STRING;
+			string = content;
+		}
+	}
+	
+	public boolean isBlank() {
+		if(wasFormula)
+			return false;
```

Comment:
I know that there were already many code-style issues present before, but how about following google's style guide (https://google.github.io/styleguide/javaguide.html#s4.1-braces), meaning that we always use braces for conditionals?